src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -61,9 +95,9 @@ private void UserControl_DataContextChanged(object sender, DependencyPropertyCha
         {
             if (e.OldValue is ReadmePreviewViewModel oldMetadata)
             {
-                ThreadHelper.JoinableTaskFactory.Run(async () =>
+                _ = NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
If I'm reading code correctly `NuGetUIThreadHelper.JoinableTaskFactory` is wrapping ThreadHelper.JoinableTaskFactory, if so, then:

[❌ AVOID calling JoinableTaskFactory.RunAsync from the default factory on fire and forget tasks](https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/24027/Threading?anchor=%E2%9D%8C-**avoid**-calling-%60joinabletaskfactory.runasync%60-from-the-default-factory-on-fire-and-forget-tasks).

Usually this is wrapped around a JoinableTaskFactory that represents the lifetime of the thing that owns the task, in this case the UI, but that's not the case, so this does actually nothing as no one ever waits on this from the UI thread.

Reply:
I switched it back to "Run" for now